include/odp/api/spec/schedule.h

```diff
@@ -45,6 +45,110 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration
+ * When schedule configuration is not called by the application, the scheduler
+ * behaves in flow-unaware mode and event flow hash values are ignored
+ */
+typedef struct odp_schedule_config_t {
+	/** Number of flows per queue to be supported
+	 * Scheduler enables flow aware mode when flow count is configured
+	 * greater than 1.
+	 *
+	 * Flows are lightweight entities and packets can be assigned to
+	 * specific flows by the application using odp_event_flow_hash_set()
+	 * before enqueuing the packet into the scheduler. This value is
+	 * ignored unless scheduler supports flow aware mode.
+	 *
+	 * Depening on the implementation this number might be rounded up to
+	 * nearest supported value (e.g power of 2). odp_schedule_config() will
+	 * update passed config with the value actually used. This number
+	 * should be less than maximum flow supported by the implementation.
+	 *
+	 * @see odp_schedule_capability_t
+	 */
+	uint32_t flow_count;
+
+	/** Maximum number of schedule queues to be supported
+	 * Application configures the maximum number of schedule queues to be
+	 * supported by the implementation.
+	 * @see odp_queue_capability_t
+	 */
+	uint32_t queue_count;
```

Comment:
1) count => num: num_queues
2) Maximum number of scheduled queues to be used. (The second sentence is not needed)
3) queue capability three max_nums: blocking, non-blocking lockfree and waitfree. This should contain the same information (so that scheduler capa can be removed from queue API)